### ISSUE TITLE
The story on stable config options

### ISSUE THREAD

**Issue by softprops** [rust-lang-nursery/rustfmt#2690]
I'm a huge fan of rustfmt and there is only one deviation from the defaults that I tried to use. I like my args vertical.

https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#fn_args_density

Unfortunately this option flagged as unstable. I saw the note in the readme about when to expect these to work based on toolchains. I try to stay on stable by default I found this problematic. I use the vscode rls extension which formats srcs using rustfmt on save but that won't honor my rustfmt.toml because I compile with stable by default.

I was curious if these stable markers were based on technical vs community agreement constraints.

**Comment by nrc** [comment]
Stabilisation of options is basically a technical thing - when we are sure that the option is a good long-term proposition, i.e., is designed well, pulls its weight, fits nicely in the implementation, etc. The defaults have been decided by an RFC process.

**Comment by softprops** [comment]
Cool. I figured the defaults we're set on this one. I just wasn't sure when I could start using it on the stable rust channel. Some rust std lib unstable features have an associated tracking issue. It may be helpful for unstable rustfmt config options have them too. That would allow me to get notified when but becomes stable. I could just subscribe to the associated gh issue.

Thanks for the context though. Closing this for now. My question was answered :)
